src/sentry/options/defaults.py

```diff
@@ -1061,6 +1061,15 @@
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
 
+# Option to control whether or not we raise ValidationErrors in the indexer
+# (Temporary) raising the error would mean we skip the processing or DLQing of these
+# invalid messages
+register(
```

Comment:
Just checking, is it ok to register this in the same deploy as code that uses it now? I remember in the past it needed to be registered separately first.

Reply:
ah right, I can separate that out